### PR TITLE
VAR-153 | Make event handlers for FullCalendar

### PR DESCRIPTION
**Pull request by rBoost**

### CODE REVIEW

File: app/pages/resource/_resource-page.scss [hunk 1]
```diff
@@ -138,50 +143,32 @@ $content-padding: 10px;
...
-      .app-ResourcePage__content {
-         padding: 0px;
-      }
-   }
+    #footer {
+      position: relative;
+      z-index: -15;
+    }
+    .app-ResourceHeader__content {
+      padding: 15px 15px 15px 15px;
```
Comment (midori0507) [inline app/pages/resource/_resource-page.scss]: `padding: 15px` ?

File: src/domain/resource/reservationCalendar/ResourceReservationCalendar.js [hunk 2]
```diff
@@ -100,50 +145,212 @@ class ResourceReservationCalendar extends React.Component {
 
     const momentDate = moment(date);
     const activeStart = moment(info.view.activeStart);
-    const activeEnd = moment(info.view.activeEnd);
+
+    // For some weird reason the activeEnd date is always last day of the day/week view + 1
+    // (e.g. for a week view it's always the next weeks monday);
+    const activeEnd = moment(info.view.activeEnd).subtract(1);
```
Comment (Nizar-Rahme) [inline src/domain/resource/reservationCalendar/ResourceReservationCalendar.js]:
```suggestion
    const activeEnd = moment(info.view.activeEnd).subtract(1, 'day');
```

File: app/pages/resource/_resource-page.scss (hunk 1 above)
Comment (rBoost) [reply app/pages/resource/_resource-page.scss]: not my style mate :)

Comment (midori0507) [reply app/pages/resource/_resource-page.scss]: Its same tho, but shorter :D

Comment (rBoost) [reply app/pages/resource/_resource-page.scss]: I just did some cleaning up for the existing styles. I'm not going to start fixing every issue in them :)

File: src/domain/resource/reservationCalendar/ResourceReservationCalendar.js [hunk 3]
```diff
@@ -100,50 +145,212 @@ class ResourceReservationCalendar extends React.Component {
...
+              </strong>
+              {' '}
+              <span className="app-ResourceReservationCalendar__selectedDateValue">
+                {this.getSelectedDateText()}
+              </span>
+            </div>
+            <Button
+              bsStyle="primary"
+              className="app-ResourceReservationCalendar__reserveButton"
+              onClick={() => this.onReserveButtonClick()}
```
Comment (Nizar-Rahme) [inline src/domain/resource/reservationCalendar/ResourceReservationCalendar.js]:
```suggestion
              onClick={this.onReserveButtonClick}
```

**Pull request merged**